### PR TITLE
Updated self-hosted index page

### PR DESCRIPTION
**Pull request by MimiDumpling**
The Self-Hosted install page was recently updated. I went through and cleaned up some spelling/grammar/language.

### CODE REVIEW

File: src/collections/_documentation/server/installation/index.md [hunk 1]
```diff
@@ -3,34 +3,34 @@ title: Installation
...
+Before starting the installation, we strongly recommend you to check out [how to configure your Sentry instance]({%- link _documentation/server/config.md -%}) as you'd need to rebuild your images (`docker-compose build`) if you ever want to change your configuration settings. You may copy and edit the example configs provided in the repository. If none exists, the install script will use these examples as the actual configurations.
 
-To start, just run the install script:
+To start, run the install script:
 ```bash
 ./install.sh
 ```
 
-If the `CI` environment variable is set to a non-empty value, the script will assume unattended installation and will skip some steps. You can look at [its source code](https://github.com/getsentry/onpremise/blob/master/install.sh) to figure out what those steps are, which will also be logged to standard output.
+If you've set the `CI` environment variable to a non-empty value, the script will assume unintended installation and skip some steps. You can look at [its source code](https://github.com/getsentry/onpremise/blob/master/install.sh) to figure out what those steps are, which will also be logged to standard output.
```
Comment (BYK) [inline src/collections/_documentation/server/installation/index.md]: Let's revert the changes on this paragraph as it changes the meaning. The original one was accurate:

1. The `CI` environment variable is usually set by continuous-integration services such as TravisCI. We don't expect users to set this manually as the new wording suggests (nor we want to encourage this).
2. The script will assume _unattended_ installation, meaning this is fully automated and no keyboard input will be expected (and probably won't be available anyway).

Comment (MimiDumpling) [reply src/collections/_documentation/server/installation/index.md]: @BYK what if we said, "If your continuous-integration service sets the `CI` environment variable to a non-empty value, the script will assume unattended installation and will skip some steps."

Comment (BYK) [reply src/collections/_documentation/server/installation/index.md]: As discussed over Slack, we want to keep the dual meaning here: 
1. This variable is usually automatically set by services like TravisCI, so we want to inform you what will happen in that case
2. We also want to highlight the possibility of using `CI=1 ./install.sh` for an on-purpose unattended install locally or on a different environment.

Comment (MimiDumpling) [reply src/collections/_documentation/server/installation/index.md]: Yup yup! You right. 😄

**Pull request merged**